lib/ja_serializer/formatter/utils.ex

```diff
@@ -60,6 +60,14 @@ defmodule JaSerializer.Formatter.Utils do
   @doc false
   def do_format_key(key, :underscored), do: key
   def do_format_key(key, :dasherized), do: String.replace(key, "_", "-")
+
+  def do_format_key(key, :camelcased) do
+    camelized = Macro.camelize(key)
```

Comment:
I wasn't familiar with this so I looked it up in the docs. The docs recommend not using this for camelizing strings since it [doesn't support unicode or characters that are not valid Elixir identifiers](https://hexdocs.pm/elixir/Macro.html#camelize/1). I'm not sure what's considered a valid Elixir identifier, but unicode should probably be supported. The JSON spec says we SHOULD NOT use non-ascii chars, but it's not a "MUST NOT" so technically unicode chars are still valid. Would you be ok changing this to something that supports any unicode chars?

Reply:
I missed it, I will check what I can replace this with.

Reply:
I feel we can use either https://github.com/nurugger07/inflex or https://github.com/sobolevn/recase. If the extra dependency is problematic, we could then merge the code with attribution.

Reply:
Ah just noticed there is Inflex already, great!